### ISSUE TITLE
Timeline panel updates

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/devtools-docs#28]
![image](https://f.cloud.github.com/assets/39191/1729735/20ab5548-62c7-11e3-830e-5cccc654550f.png)
![image](https://f.cloud.github.com/assets/39191/1729742/42758630-62c7-11e3-9e0c-2c972e760a26.png)
![image](https://f.cloud.github.com/assets/39191/1735584/68de87c8-6357-11e3-909d-7bfde4a2d0b1.png)
![image](https://f.cloud.github.com/assets/39191/1735615/0582a334-6358-11e3-8771-dfde1d2dd483.png)
- Details pane / summary.
- gpu frames on top 
- Capture stacks checkbox (will capture stack traces for all operations it can. pretty much the only part of timeline that has overhead.)
- GPU activity bars (enabled by setting)
- filters
- gpu memory in counters
- first paint, domcontentloaded, windowload, and `console.timeStamp` plotted 

Comment below with other parts that are new…

**Comment by heatheramahan** [comment]
We're addressing these updates via an HTML5Rocks article. 
I put my current draft into a google doc, which our chrome-devtools-docs group should be able to access. 

Would love to get some feedback! Thanks.

**Comment by paulirish** [comment]
yeah. This ticket is old but it made me realize some of this older stuff wasn't yet in.

all sounds good.

**Comment by heatheramahan** [comment]
ah, that's true. The H5R article covers stuff that's currently still in Canary. 

But our reference docs still might be a bit behind with respect to the items you originally listed in this ticket. I can look into that.

**Comment by madroneropaulo** [comment]
Guys, I'm not sure in which version it changed so drastically, but I couldn't find any documentation for the new timeline and frame rate debugging tools. Using latest stable chrome: 47.0.2526.106 m. While some tools are self explanatory, some aren't.

**Comment by Garbee** [comment]
It will get updated, trying to get that figured out now.

The new repository for issues is [WebFundamentals](https://github.com/Google/WebFundamentals) btw. Issue [2404](https://github.com/google/WebFundamentals/issues/2404) there already exists for tracking this update (assuming none pre-date it.)
